File: yunshin/4주차/BruteForce/Programmers/42842_카펫.js
function solution(brown, yellow) {
  heightCandidate = []
  for (let h = 0; h < brown / 2; h++) if (2 * (brown + yellow) == (brown + 4 - 2 * h) * h) heightCandidate.push(h)
  if (heightCandidate.length == 1) return [heightCandidate[0], heightCandidate[0]]
  else return heightCandidate.sort((a, b) => b - a)
}

File: yunshin/4주차/BruteForce/Programmers/42842_카펫.py
def solution(brown, yellow):
    heights = []
    for h in range(1, brown//2):
        if h * ( (brown+4)//2 - h ) ==  brown+yellow:
            heights.append(h)
    if len(heights) == 1:
        return [heights[0],heights[0]]
    else:
        return sorted(heights,reverse=True)